README.md

```diff
@@ -525,6 +525,8 @@ Some [Groupware](#groupware) solutions also feature file sharing and synchroniza
   * [Apaxy](http://adamwhitcroft.com/apaxy/) - Theme built to enhance the experience of browsing web directories, using the mod_autoindex Apache module and some CSS to override the default style of a directory listing. ([Source Code](https://github.com/AdamWhitcroft/Apaxy) `Public domain` `CSS`
   * [eXtplorer](http://extplorer.sourceforge.net/) - A PHP and JavaScript based File Manager
   * [h5ai](https://larsjung.de/h5ai/) - Modern file indexer for HTTP web servers with focus on your files. Directories are displayed in a appealing way and browsing them is enhanced by different views, a breadcrumb and a tree overview. ([Demo](https://larsjung.de/h5ai/demo/), [Source Code](https://github.com/lrsjng/h5ai)) `MIT` `PHP/Javascript`
+  * [Omeka](http://omeka.org) - Create complex narratives and share rich collections, adhering to Dublin Core standards with Omeka on your server, designed for scholars, museums, libraries, archives, and enthusiasts.. ([Demo](http://omeka.org/showcase/), [Source Code](http://omeka.org/download/)) `GNU GENERAL PUBLIC LICENSE` `Linux/Apache/MySQL/PHP`  
```

Comment:
@githubber 

 * It should be in the https://github.com/Kickball/awesome-selfhosted#content-management-systems-cms section
 * Source code link should be https://github.com/omeka/Omeka (you linked to the download page)
 * License should be written as `GPLv3`
 * Language/platform should simply be `PHP`